src/configuration/xcWebSocketBridgeConfiguration.ts

```diff
@@ -7,6 +7,7 @@ export const Kinds = {
 
 
 export enum Commands {
+    hb,
```

Comment:
To get "hb" string for example: Commands[Commands.hb] == "hb"
The enum was something like 
    var commands = {
        snapshot: "snapshot",
        update: "update",
        ...
    };
But NE requested to create an enum rather than a simple JSON.